supply-chain/audits.toml

```diff
@@ -352,6 +391,23 @@ This bump only changed from a function to an associated `const` and trivially
 contains no significant changes.
 """
 
+[[audits.miniz_oxide]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-run"
+version = "0.5.3"
+notes = "This version is used in rust's libstd, so effectively we're already trusting it"
+
+[[audits.object]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+version = "0.29.0"
+notes = "This version is used in rust's libstd, so effectively we're already trusting it"
+
+[[audits.object]]
+who = "Benoît du Garreau <bdgdlm@outlook.com>"
+criteria = "safe-to-deploy"
+delta = "0.29.0 -> 0.28.4"
```

Comment:
Is this still necessary? I think that wasmtime is using 0.29.0 so the above audit for that version should be good enough.

Reply:
`cargo vet` got confused with existing imports